packages/client-core/src/admin/components/Location/LocationTable.tsx

```diff
@@ -151,7 +151,7 @@ const LocationTable = (props: LocationProps) => {
       el.sceneId,
       el.maxUsersPerInstance,
       el.slugifiedName,
-      el.location_setting?.locationType,
+      el.location_settings?.locationType,
```

Comment:
All of these actually need to be 'location_setting', without the 's', unless something else got changed to make that work. Sequelize singularizes table names if it's only getting a single item. While the table name is 'location_settings', it's populated on the location when returned to the client as 'location_setting'.